src/services/alert.service.ts

```diff
@@ -17,6 +17,20 @@
 import { IHttpPromise } from 'angular';
 import { Alert, Scope } from '../entities/alert';
 
+// tslint:disable-next-line:interface-name
```

Comment:
Why not apply this rule?

Reply:
deleted and use interface prefixed with i